modules/installation-configuration-parameters.adoc

```diff
@@ -124,6 +128,11 @@ link:https://yaml.org/spec/1.2/spec.html#sequence//[YAML sequence].
 |Valid link:https://docs.aws.amazon.com/general/latest/gr/rande.html[AWS region],
 such as `us-east-1`
 
+|`controlPlane.replicas`
+|The number of control plane machines to provision. You must provision at least
+one control plane machine.
+|A positive integer.
```

Comment:
controlPlane.repicas is a positive integer, so `You must provision at least
￼	one control plane machine.` is not necessary?

Reply:
Because the bug: https://bugzilla.redhat.com/show_bug.cgi?id=1671136, seems for 4.1, we do not support 0 worker machines.

Reply:
@qinpingli, I'm double-checking with the installer team, but I left you a comment in the bug.